addons/project/data/project_demo.xml

```diff
@@ -758,6 +758,30 @@
             <field name="body"><![CDATA[<ul class="mb-0 list-unstyled d-flex flex-column gap-1"><li class="o-mail-Message-tracking" role="group"><span class="o-mail-Message-trackingOld text-muted fw-bold">In Progress</span><i class="o-mail-Message-trackingSeparator fa fa-long-arrow-right mx-2 text-600"></i><span class="o-mail-Message-trackingNew me-1 fw-bold text-info">Done</span><span class="o-mail-Message-trackingField ms-1 fst-italic text-muted">(Stage)</span></li></ul>]]></field>
         </record>
 
+        <record id="project.project_1_task_2" model="project.task">
+            <field name="duration_tracking"
+                eval="{
+                    'd': DateTime.now() - relativedelta(months=1),
+                    's': ref('project.project_stage_2'),
+                    ref('project.project_stage_0'): ((DateTime.now() - relativedelta(months=2)) - (DateTime.now() - relativedelta(months=5))).total_seconds(),
+                    ref('project.project_stage_1'): ((DateTime.now() - relativedelta(months=1)) - (DateTime.now() - relativedelta(months=2))).total_seconds(),
+                    'stage_history_tracking': [
```

Comment:
Why is there a `stage_history_tracking` in `duration_tracking`? I don't think we need that.

For the burndown chart, we agreed to simplify the specification. So, if you have the following events:

1. New: 1d
2. In Progress: 2d
3. New: 4d
4. In Progress 8d
5. Done: 4d

If, today, we are on Dec 31, we can show this in hte burndown chart:
- Dec 27: Done
- Dec 17: In Progress (-10d)
- Dec 12: New (-5d)

So, just using the time per stage is enough to compute this burndown chart. I know it's a bit less precise than reality, but we think it's ok. Just a dfferent behaviour measured, but still correct. (like a flamechart by Time or by Method, both are correct, just a different way to view the same data)